typedoc.js

```diff
@@ -22,9 +22,9 @@ module.exports = {
 
     '**/packages/metapackage/**'
   ],
-  excludeNotExported: true,
-  ignoreCompilerErrors: false,
-  mode: 'library',
+  // excludeNotExported: true,
+  // ignoreCompilerErrors: false,
+  // mode: 'library',
```

Comment:
you can delete this line, at least. From what I read I don't think the typedoc guy wants to keep the name "library mode" around in the new version

Reply:
> you can delete this line, at least. From what I read I don't think the typedoc guy wants to keep the name "library mode" around in the new version

From the issue linked above, I think all three of these options are not in 0.20 anymore. Thanks for the confirmation.